internal/captain/command.go

```diff
@@ -880,3 +918,30 @@ func childCommands(cmd *Command) string {
 
 	return fmt.Sprintf("\n\nAvailable Commands:\n%s", table.Render())
 }
+
+func (c *Command) LogArgs() {
+	children, err := c.FindChildren(os.Args[1:])
```

Comment:
We should use `c.Args`. I think for those it already dropped the first entry of `os.Args`, but worth verifying.